test/src/tests-persistence.ts

```diff
@@ -145,9 +153,10 @@ describe('persistence', () => {
     });
 
     it('localStorage - should key cookies on mpid on first run', done => {
-        mParticle._resetForTests(MPConfig);
```

Comment:
Why is this being removed?

Reply:
there is no init in the beginning, so it was not needed